app/controllers/admin/provisioning_keys_controller.rb

```diff
@@ -0,0 +1,35 @@
+class Admin::ProvisioningKeysController < Admin::AdminController
+
+  def index
+    @provisioning_keys = ProvisioningKey.all
+  end
+
+  def show
+    @provisioning_key = ProvisioningKey.find(params[:id])
```

Comment:
You can move the loading of the instance variable into a private method:

```
def set_provisioning_key
  @provisioning_key = ProvisioningKey.find(params[:id])
end
```

This can be called using a `before_action` at the top of the controller:

`before_action :set_provisioning_key, only: [:show, :destroy]`